include/swift/AST/DiagnosticsSIL.def

```diff
@@ -979,31 +979,38 @@ NOTE(regionbasedisolation_isolated_since_in_same_region_basename, none,
 //
 
 ERROR(regionbasedisolation_named_transfer_yields_race, none,
-      "transferring %0 may cause a data race",
+      "sending %0 risks causing data races",
       (Identifier))
 
 NOTE(regionbasedisolation_named_info_transfer_yields_race, none,
-     "transferring %1 %0 to %2 callee could cause races in between callee %2 and local %3 uses",
+     "sending %1%0 to %2 callee risks causing data races between %2 and local %3 uses",
      (Identifier, StringRef, ActorIsolation, ActorIsolation))
+NOTE(regionbasedisolation_named_info_transfer_yields_race_callee, none,
+     "sending %1%0 to %2 %3 %4 risks causing data races between %2 and local %5 uses",
+     (Identifier, StringRef, ActorIsolation, DescriptiveDeclKind, DeclName, ActorIsolation))
+
 NOTE(regionbasedisolation_named_transfer_non_transferrable, none,
-      "transferring %1 %0 to %2 callee could cause races between %2 and %1 uses",
-      (Identifier, StringRef, ActorIsolation))
+      "sending %1%0 to %2 callee risks causing data races between %2 and %3 uses",
+      (Identifier, StringRef, ActorIsolation, StringRef))
+NOTE(regionbasedisolation_named_transfer_non_transferrable_callee, none,
+      "sending %1%0 to %2 %3 %4 risks causing data races between %2 and %5 uses",
+      (Identifier, StringRef, ActorIsolation, DescriptiveDeclKind, DeclName, StringRef))
 NOTE(regionbasedisolation_named_transfer_into_transferring_param, none,
-     "%0 %1 is passed as a transferring parameter; Uses in callee may race with later %0 uses",
+     "%0%1 is passed as a transferring parameter; Uses in callee may race with later %0uses",
      (StringRef, Identifier))
 NOTE(regionbasedisolation_named_notransfer_transfer_into_result, none,
-     "%0 %1 cannot be a transferring result. %0 uses may race with caller uses",
-     (StringRef, Identifier))
+     "%0%1 cannot be a transferring result. %2 uses may race with caller uses",
```

Comment:
This message still says "transferring result"

Reply:
(`transferring` isn't renamed yet)

Reply:
This is also since we haven't renamed transferring -> sending.